src/Compatibility/ApiCompat/Microsoft.DotNet.PackageValidation/Validators/BaselinePackageValidator.cs

```diff
@@ -117,6 +117,15 @@ public void Validate(PackageValidatorOption options)
                 }
             }
 
+            // If baseline target frameworks were ignored, log them as an informational message.
+            if (options.BaselinePackageFrameworkFilter is not null &&
+                options.BaselinePackageFrameworkFilter.FoundExcludedTargetFrameworks.Count > 0)
+            {
+                log.LogMessage(ApiSymbolExtensions.Logging.MessageImportance.Normal,
```

Comment:
Consider a higher level of importance if we aren't emitting any suppressions.  Try to increase the visibility of the omitted testing.

Is there anything more we might want to do to catch a case where someone might accidentally omit comparisons?

Reply:
`Normal` message importance is the highest we can go here. We don't ever want to emit messages with `High` importance outside of a failure path.

Reply:
I see.  If all frameworks are excluded should we emit a warning?  If frameworks are excluded from the baseline that are present explicitly in the new package should we emit a warning?  Just brainstorming some potential pitfalls here and how we might mitigate them.  If you think these are good suggestions I'd be OK with us handling those or others separately.

Reply:
I thought I replied to that yesterday. Yes that's a great idea.